scripts/fetch-docs.sh

```diff
@@ -0,0 +1,21 @@
+mkdir -p resources/docs 
+#  Bitbake docs
+git clone --depth 1 --filter=blob:none --sparse https://github.com/openembedded/bitbake.git 
+cd bitbake 
+git sparse-checkout set doc/bitbake-user-manual/ 
+cd doc/bitbake-user-manual/ 
+mv bitbake-user-manual-metadata.rst bitbake-user-manual-ref-variables.rst  ../../../resources/docs 
+cd ../../../ 
+rm -rf bitbake
```

Comment:
The temporary directories should be gitignored. I think we should also keep them and remove only with `npm run clean`